barberia/src/App/pages/Calendar.js

```diff
@@ -1,20 +1,28 @@
 import React from "react";
 import DatePicker from "react-datepicker";
 import AppoPicker from "./TimePicker";
-
+import {A} from "hookrouter";
 import "react-datepicker/dist/react-datepicker.css";
 
-function Calendar() {
-  const [startDate, setStartDate] = React.useState(new Date());
+function Calendar({formDate,setFormDate}) {
+  const [startDate, setStartDate] = React.useState(formDate || new Date());
+  console.log("date is ",formDate ,setFormDate);
```

Comment:
Please can you remove this `console.log`. It's good practice not to leave stray `console.logs` in Production code.

Reply:
cleaned